apps/etterlatte-saksbehandling-ui/client/src/components/behandling/trygdetid/TrygdetidIAnnenBehandlingMedSammeAvdoede.tsx

```diff
@@ -0,0 +1,136 @@
+  const [skjult, setSkjult] = useState(false)
+  const [trygdetidKopiert, setTrygdetidKopiert] = useState(false)
+
+  const kopierTrygdetid = (kildeBehandlingId: string) => {
+    trackClick('kopier-trygdetidsgrunnlag-fra-behandling-med-samme-avdoede')
+    kopierTrygdetidReq(
+      {
+        behandlingId: behandlingId,
+        kildeBehandlingId: kildeBehandlingId,
+      },
+      (trygdetider) => {
+        setTrygdetidKopiert(true)
+        setTrygdetider(trygdetider)
+      }
+    )
+  }
+
+  useEffect(() => {
+    hentKandidatForKopieringAvTrygdetidReq(behandlingId)
+  }, [])
+
+  useEffect(() => {
+    setVisDetaljer(!harRedigertTrygdetidGrunnlag())
+    setSkjult(trygdetidKopiert)
+    setTrygdetidKopiert(false)
+  }, [trygdetider])
+
+  if (skjult) {
+    return <></>
+  }
+
+  return (
+    <>
+      {mapResult(hentKandidatForKopieringAvTrygdetidStatus, {
+        success: (behandlingId) => (
+          <>
+            {behandlingId ? (
+              <Box paddingBlock="2">
+                <Alert variant="info">
+                  <HStack gap="6" justify="space-between">
+                    <Heading size="small" level="2">
+                      Det finnes en annen behandling tilknyttet samme avdøde
+                    </Heading>
+                    {!visDetaljer && (
+                      <Link href="#" onClick={() => setVisDetaljer(true)}>
+                        Les mer
+                      </Link>
+                    )}
+                  </HStack>
+                  {visDetaljer && (
+                    <VStack gap="3">
+                      <BodyShort>
+                        Det finnes en annen behandling tilknyttet avdøde {avdoedesNavn()}, der trygdetiden allerede er
+                        fylt inn. Ønsker du å benytte den samme trygdetiden i denne behandlingen? Dette overskriver det
+                        du eventuelt har registrert allerede.
+                      </BodyShort>
+                      <Link href={`/behandling/${behandlingId}/trygdetid`} as="a" target="_blank">
+                        Gå til behandlingen
+                        <ExternalLinkIcon>Gå til behandlingen</ExternalLinkIcon>
+                      </Link>
+                      <HStack gap="4" justify="space-between">
+                        <Button
+                          variant="primary"
+                          icon={<PlusCircleIcon fontSize="1.5rem" />}
+                          onClick={() => kopierTrygdetid(behandlingId)}
+                        >
+                          Ja, kopier og legg til trygdetid
+                        </Button>
+                        <Button
+                          variant="secondary"
+                          icon={<XMarkOctagonIcon fontSize="1.5rem" />}
+                          onClick={() => setVisDetaljer(false)}
+                        >
+                          Nei, jeg ønsker å fylle ut manuelt
+                        </Button>
```

Comment:
Endre rekkefølge på disse for å følge resten av Gjenny. 

Høyre skal alltid være den bekreftende handlingen